### ISSUE TITLE
Log level not being fully applied

### ISSUE THREAD

**Issue by josepplaltg** [anchore/anchore-engine#155]
**Is this a request for help?**: 
NO
---

**Is this a BUG REPORT or a FEATURE REQUEST?** (choose one):
BUG REPORT

**Version of Anchore Engine and Anchore CLI if applicable**:
anchore-cli version 0.2.4
anchore-engine version 0.3.2
 
**What happened**:
Setting the log level to ERROR on anchore-engine config.yaml don't set the log level for Twisted, then you still get all the simplequeue request logs on STDOUT.

**What did you expect to happen**:
Only get ERROR logs

**Any relevant log output from /var/log/anchore**:
[service:simplequeue] 2019-03-11 16:20:25+0000 [-] "10.11.15.69" - - [11/Mar/2019:16:20:25 +0000] "GET /v1/leases/analyzer_queue/release?client_id=anchore-engine-0%3A13%3A140150475228928%3A&epoch=13753874 HTTP/1.1" 200 - "-" "python-requests/2.20.0"
[service:simplequeue] 2019-03-11 16:20:27+0000 [-] "10.11.15.69" - - [11/Mar/2019:16:20:26 +0000] "POST /v1/queues/watcher_tasks/is_inqueue HTTP/1.1" 200 3 "-" "python-requests/2.20.0"

**What docker images are you using**:
docker.io/anchore/anchore-engine:v0.3.2

**How to reproduce the issue**:
Deploy anchore engine 0.3.2 and set log_level: ERROR on config.yaml file, you will see logs like previous ones

**Anything else we need to know**:

**Comment by nurmi** [comment]
We've added a new configuration parameter 'enable_access_logging' which, when set to 'False' in anchore-engine's config.yaml (either globally or as an override, per service section) will prevent the twisted access logs from appearing in the anchore engine log output.  

background: these particular access logs in twisted take a different path (which uses a very low level simple 'write' that is not processed through normal logging paths) than regular log level controlled log entries, and thus we opted to control them separately from the regular log level controlled information.  In addition, we felt that one may wish to view the access logs independent from the regular anchore log level controlled entries, which this solution permits.

This should address the core issue (happy to discuss any feedback on the solution, here!).  The option will be included in the next release of anchore engine (0.3.4) and versions moving forward.
